game/scripts/npc/abilities/faceless_void_time_lock_oaa.txt

```diff
@@ -41,12 +41,12 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "chance_pct"                                      "10 15 20 25 25 25"
+        "chance_pct"                                      "25"
```

Comment:
I don't like seeing the chance buffed here, but I'm not opposed to a void buff. I just think it's the wrong part of him that i'd like to see become more powerful.

Reply:
This buff is fine, makes his early game stronger. But if you actually want to make void good without changing time lock, increase his agi gain and lower his BAT.

Reply:
I think agi gain might be hard-coded either way I wouldn't be the one capable on knowing how to do that at this moment.

Reply:
Jakiro had is agi gain buffed.